Assets/Scripts/Ability/IndicatorController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.Rendering.Universal;
+
+namespace RogueApeStudio.Crusader.Ability
+{
+    public class IndicatorController : MonoBehaviour
+    {
+        [SerializeField] private Transform _pivot;
+        [SerializeField] private Camera _cam;
+        [SerializeField] private PlayerInput _playerInput;
+        [SerializeField] private LayerMask _layerMask;
+        [SerializeField] private float _offset;
+        private CastTarget _castTarget;
+
+        void Update()
```

Comment:
Does this need to happen every frame? Do you disable this component? If so, why is there no `Disable()` and `Enable()` functions.

Reply:
needs to happen every frame because of player movement and mouse movement, the component does get disabled, no `Disable()` or `Enable()`, because they would be empty.

Reply:
done